### DISCUSSION TITLE
Clean up folder hierarchy

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [WohlSoft/PGE-Project#597] [Q&A]
What about cleaning up the folder hierarchy?

```
\ MoonDust
-\ Assets
--\Editor
---\Plugins
----\Gfx2Png
----|Gfx2Png.dll
--\Engine
---\Games
----\TheXTech
-----\Assets
------|Textures
------|Fonts
------|Music
------|Sound
------|Settings
------|Languages
------|Logs
-----\Worlds
-----|Battle
```

**Comment by Wohlstand** [comment]
I thinking about reorganising of the hieracly during the future rebuilding. Anyway, the final way of the hieracly organising will be designed when actual rebuilding will be started. Also, I guess, this post could be a discussion rather than an issue.
